README.md

```diff
@@ -145,6 +147,17 @@ Any failures potentially indicate an issue with the set-up of the Rest API or co
     ```
 After the uninstallation process completes successfully, the product no longer contains the plug-in configuration.
 
+## Troubleshooting
+
+### The IBM Db2 Database Plug-in for Zowe CLI is no longer compatible with the installed Node.js version.
+
```

Comment:
I'm thinking you may be burying the lede here. Would something like this work?:
**Incompatibility with Node.js version**, or **Node.js version incompatible with plug-in**